libs/model/src/thread/GetThreads.query.ts

```diff
@@ -252,37 +250,24 @@ export function GetThreads(): Query<typeof schemas.GetThreads> {
         },
       );
 
-      const countThreadsQuery = models.sequelize.query<{ count: number }>(
-        `
-          SELECT COUNT(*) AS count
-          FROM "Threads" T
-          WHERE ${baseWhereClause}
-        `,
-        {
-          replacements,
-          type: QueryTypes.SELECT,
-        },
-      );
-
       const numVotingThreadsQuery = models.Thread.count({
         where: {
           community_id,
           stage: 'voting',
         },
       });
 
-      const [threads, numVotingThreads, countResult] = await Promise.all([
+      const [threads, numVotingThreads] = await Promise.all([
         responseThreadsQuery,
         numVotingThreadsQuery,
-        countThreadsQuery,
       ]);
 
       return {
         limit: replacements.limit,
         page: replacements.page,
         threads,
         numVotingThreads,
-        threadCount: Number(countResult[0]?.count) || 0,
+        threadCount: threads.at(0)?.total_num_thread_results || 0,
```

Comment:
i think this was to be the total number of threads in the topic without any filters.

Reply:
It is. But the postgres payload attaches the same total count to all threads for simplicity.

Reply:
@rbennettcw won't `total_num_thread_results` be limited by the `LIMIT` clause?